src/constants/fonts/test.ts

```diff
@@ -0,0 +1,10 @@
+import PRIMARY_FONTS from './primary';
+import SECONDARY_FONTS from './secondary';
+
+describe('theme fonts', () => {
+  it('primary and secondary fonts have the same number of properties', () => {
+    expect(Object.keys(PRIMARY_FONTS).length).toEqual(
```

Comment:
While we're here and adding more of these, do you think we should create a helper util to just compare the number of keys of two provided objects? It's a little cute--it's not a big deal to have to write this out every time--but it makes sense to couple some of our test scaffolding around this.

Reply:
yep added here https://github.com/curology/radiance-ui/pull/509/commits/837c9c58c0fe600926d923c415dc2643f04ea235